features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```
@@ -650,8 +650,8 @@
   "console.ui.app_copyright": "WSO2 Identity Server",
   "console.ui.app_title": "Console | WSO2 Identity Server",
   "console.ui.app_name": "Console",
-  "console.ui.app_logo_path": "/assets/images/logo.svg",
-  "console.ui.product_name": "Identity Server",
+  "console.ui.app_logo_path": "/assets/images/logo-full.svg",
```

Let's use `logo.svg` here and refactor the login portals to stop using text to resolve product name. 
```suggestion
  "console.ui.app_logo_path": "/assets/images/logo.svg",
```

---

+1 resolved in commit bcc0138